packages/orchestration/src/utils/zoe.js

```diff
@@ -0,0 +1,56 @@
+import { Fail } from '@endo/errors';
+import { atomicTransfer } from '@agoric/zoe/src/contractSupport/index.js';
+
+/**
+ * @import {ChainAddress} from '../types.js';
+ * @import {OrchestrationAccount} from '../orchestration-api.js'
+ */
+
+export const makeZoeTools = (zone, { zcf, vowTools }) => {
+  /**
+   * Transfer the `give` a seat to a local account.
+   *
+   * @param {ZCFSeat} srcSeat
+   * @param {OrchestrationAccount<any>} localAccount
+   * @param {AmountKeywordRecord} give
+   * @returns {Promise<void>}
+   */
+  const localTransfer = vowTools.retriable(
+    zone,
+    'localTransfer',
+    async (srcSeat, localAccount, give) => {
+      !srcSeat.hasExited() || Fail`The seat cannot have exited.`;
+      const { zcfSeat: tempSeat, userSeat: userSeatP } = zcf.makeEmptySeatKit();
+      const userSeat = await userSeatP;
+      atomicTransfer(zcf, srcSeat, tempSeat, give);
+      tempSeat.exit();
+      // TODO get the userSeat into baggage so it's at least recoverable
```

Comment:
What's stopping us from doing this now?

Reply:
there's no specific impediment. the TODO is because it's not done. Are you requesting that it be done before any of this lands?

Reply:
I was more so curious, but if it's not a lot of effort it would be nice to see in this PR. If not, a ticket number so we remember to come back to it would be helpful

Reply:
There's some design to figure out wrt subZone